### PR TITLE
Artist UI- new component for artist list

### LINKED ISSUE #890: artist list ui problem
In the artist list, the mobile UI is not optimized :>

Screenshot from 2021-03-23 10-17-08

should create a new UI for the displaying of artists.

### PR DESCRIPTION
**Pull request by armedev**
fixes #890 

created a new component for artists list view and wrapped the artist name.

Screenshot from 2021-03-23 11-50-50

### CODE REVIEW

File: ui/src/common/index.js [hunk 1]
```diff
@@ -27,3 +27,4 @@ export * from './useToggleStar'
 export * from './useTraceUpdate'
 export * from './Writable'
 export * from './SongSimpleList'
+export * from './artistSimpleList'
```
Comment (deluan) [inline ui/src/common/index.js:30]: Please rename the file to `ArtistSimpleList` (to keep consistency: PascalCase for classes, camelCase for functions).

**Review by deluan — CHANGES_REQUESTED** [review]
LGTM, just a minor nitpick with the file name. I'll do a throughly test before merging. Thanks!

File: ui/src/common/index.js (hunk 1 above)
Comment (armedev) [reply ui/src/common/index.js]: ok

**Review by deluan — APPROVED** [review]
LGTM. Thanks

**Pull request merged**